src/Bigtable/Connection/TableConnectionInterface.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/*
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable;
+
+use Google\Cloud\Bigtable\ConnectionInterface;
+
+/**
+ * Connection to Bigtable Table API
+ */
+interface TableConnectionInterface extends ConnectionInterface
```

Comment:
I think that for the first PR, it would be great to flesh out ideas for `readRow` and `mutateRow`.  We can flesh out the other APIs later.